pyproject.toml

```diff
@@ -39,11 +39,15 @@ dependencies = [
 "Documentation" = "https://nomad-coe.github.io/nomad-simulations/"
 
 [project.optional-dependencies]
+md = [
```

Comment:
not sure why this is an optional dep

Reply:
it is related to trying to enable easier stand-alone use of the parsers. If someone wants to just use vasp eg they need to install MDAnalysis (both for the parser dep and for nomad-simulations). So, I made the MDAnalysis optional in both packages...is it not a good approach? Do you have an alternative suggestion?

Reply:
Ah I see then ignore related comments

Reply:
I have adjusted the strategy by not throwing the import error, and instead using the flag to exit out of functions using the dependency. Let me know what you think about this